src/helm/benchmark/static/index.html

```diff
@@ -47,6 +47,15 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/js-yaml/4.1.0/js-yaml.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/showdown/2.0.3/showdown.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.7.7/handlebars.min.js"></script>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-T0MW28MP3W"></script>
```

Comment:
We want to send analytics to Google _only_ from the production website, not from from localhost or other people's websites. I would suggest commenting this out, and then we can uncomment it when we deploy.

Reply:
Perhaps it suffices to check hostname ?

Reply:
I will add a hostname check instead, so we don't need to do anything further. If you'd prefer commenting though, I can do that.

Reply:
I would prefer commenting out, because some users will be sensitive about sending data (by loading the script) to Google from on-prem deployments.

Reply:
But then we have to uncomment when we deploy (which seems non-ideal).  Is there a particular reason we need analytics right now?

Reply:
It would be helpful to get some insights on session time per page to get ideas for visual redesign. Later if we have two versions (react vs jquery, or leaderboard vs verbose results) data on that will be helpful as well,

Reply:
We already have to make various manual changes for deployment, so this isn't an significant issue.